lib/workers/repository/onboarding/pr/index.ts

```diff
@@ -174,6 +174,7 @@ If you need any further assistance then you can also [request help here](${
       const pr = await platform.createPr({
         sourceBranch: config.onboardingBranch!,
         targetBranch: config.defaultBranch!,
+        forkModeAllowMaintainerEdits: config.forkModeAllowMaintainerEdits,
```

Comment:
@rarkins i think we should pass this via the platform options below?

Reply:
yeah, I was wondering too (https://github.com/renovatebot/renovate/pull/19771#issuecomment-1426039302). 

will make the change based on what you decide

Reply:
friendly ping so I can make all the suggested changes

Reply:
Yes, please pass it as a platform option